src/app/portfolio/portfolio.component.html

```diff
@@ -1,13 +1,14 @@
 <article class="portfolio-page">
-    <h2 class="heading">COLLABORATION</h2>
-    <hr class="seperator" />
+    <h2 class="heading">COLLABS</h2><br />
+    <!-- <hr class="seperator" /><br /> -->
     <div class="collab-container">
-        <img src="../../assets/sheetal-koskii.jpg" alt="image, for koskii collaboration link" class="collab-image"
-            (click)="onClick($event)">
-        <div class="collab-content">
-            <div class="collab-text">
-                KOSKII <br>
-                <hr class="link-line" />
+        <div class="collab-card" *ngFor="let image of collabData">
+            <img src="{{image.url}}" alt="image, for koskii collaboration link" class="collab-image"
```

Comment:
<img> tag shouldn't have click listeners on it. It doesn't make the experience accessible.